orttraining/orttraining/python/ort_trainer.py

```diff
@@ -681,6 +628,9 @@ def _init_session(self):
         if self.onnx_model_ is None:
             return
 
+        if self._enable_internal_postprocess:
```

Comment:
This postprocess cover the scenario where user provides onnx_model from the beginning